### DISCUSSION TITLE
How Dynamically ComboBox Title Create

### DISCUSSION THREAD

**Discussion by BEDIRHANSAMSA** [HandyOrg/HandyControl#1007] [Q&A] [answered]
Hi,

I'am creating Dynamically Combobox how can I set hc:InfoElement.Title?

![image](https://user-images.githubusercontent.com/44173331/136200013-c0f96b70-e1d4-4774-a379-5b85ce8f1162.png)
![image](https://user-images.githubusercontent.com/44173331/136200068-68e76b79-b1d7-495c-b4f4-6d7dc55cf578.png)

**Comment by BEDIRHANSAMSA** [comment, marked as answer]
@ghost1372 Can you help?

**Reply by ghost1372** [reply]
Since the title is an AttachedProperty, you should do this:
`InfoElement.SetTitle(cmbBox, "my title");`

**Reply by BEDIRHANSAMSA** [reply]
Thank you very much :)
![image](https://user-images.githubusercontent.com/44173331/136203183-5dd69c56-f05c-431a-a394-49ab8b1a499f.png)
